R/tider_correct.R

```diff
@@ -17,22 +19,26 @@
 #' @param s2_ctrl A numeric of the Prinicipal solor semidiurnal constituent from tAmatchhe nearest NOAA
 #'                control station.
 #' @param mn_ctrl A numeric of the Mean range of tide from nearest NOAA control station
+#' @param unit A character of the units of the water level data
 #' @param type the corrected value to return
-#' @return returns a numeric of the mean high water spring or neap
+#' @return returns a character of the mean high water spring or neap with units
```

Comment:
A function like this really should return a numeric and not a character.  Units are usually something that should be tracked by the user (preferably within a script) and not dealt with inside of the function.  Before I pull this in, I think it is probably best to keep these as numeric.  Adding in something about the units to the descrition (i.e. something along the lines of "Its the users responsibility" would be fine.